dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -149,6 +149,13 @@ paths:
             Set to false to improve the query performance. However, the response will not contain
             fields startTime, failures, failureReason.
             This parameter is forwarded to Elasticsearch/Opensearch /_snapshot/ API.
+        - in: query
+          name: pattern
+          schema:
+            type: string
+            default: '*'
+          description: |
+            Provide a possible prefix number for filtering the backups, can end with a wildcard('*').
```

Comment:
Thanks for spotting this!

:wrench: I would also add that since the backupId is a number, then the matcher can only contains numbers and the wildcard